TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/ChangeLog.cshtml.cs

```diff
@@ -40,6 +46,47 @@ public async Task<IActionResult> OnGet()
 
         var notesResult = await crmQueryDispatcher.ExecuteQuery(new GetNotesByContactIdQuery(PersonId));
 
+        using var dbContext = await dbContextFactory.CreateDbContextAsync();
+        var personIdString = PersonId.ToString();
+        var eventsWithUser = await dbContext.Database
+            .SqlQuery<EventWithUser>($@"
```

Comment:
Nit: could use `"""` here to fix the weird indenting

Reply:
ah raw string literals - nice!